src/docker-outside-of-docker/install.sh

```diff
@@ -196,14 +196,14 @@ else
         apt-get -y install --no-install-recommends moby-cli${cli_version_suffix} moby-buildx
         apt-get -y install --no-install-recommends moby-compose || echo "(*) Package moby-compose (Docker Compose v2) not available for OS ${ID} ${VERSION_CODENAME} (${architecture}). Skipping."
     else
-        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix}
+        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix} docker-buildx-plugin docker-compose-plugin
```

Comment:
Can you bump the patch version for this Feature?
https://github.com/devcontainers/features/blob/d20c30770746c2d88553d3aca54d433a02ab025a/src/docker-outside-of-docker/devcontainer-feature.json#L3